tests/search/components/TestSearchResults.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import SearchResults from 'search/components/SearchResults';
+
+const render = ReactTestUtils.renderIntoDocument;
+const isDOMComponent = ReactTestUtils.isDOMComponent;
+
+
+describe('<SearchResults />', () => {
+  function renderResults(props) {
+    return render(<SearchResults {...props} />);
+  }
+
+  it('renders empty search results container', () => {
+    const root = renderResults();
+    const searchResults = root.refs.container;
+    assert.ok(isDOMComponent(searchResults));
+    assert.equal(searchResults.childNodes.length, 0);
+  });
+
+  it('renders error when query is an empty string', () => {
+    const root = renderResults({query: ''});
+    const searchResultsMsg = root.refs.message;
+    assert.include(searchResultsMsg.firstChild.nodeValue, 'supply a valid search');
```

Comment:
Doesn't need to since there's no active content so no implicit spans added by react.